tests/TestCase/ORM/TableTest.php

```diff
@@ -26,6 +26,7 @@
 use Cake\Event\Event;
 use Cake\Event\EventManager;
 use Cake\I18n\Time;
+use Cake\ORM\Association\BelongsTo;
```

Comment:
Gosh is that annoying - locally everything is fine:

    Registering sniffs in the CakePHP Core standard... DONE (99 sniffs registered)
    ... (nothing found)

But stickler reports it as wrong order?